client/routes/pageNotFound.js

```diff
@@ -0,0 +1,27 @@
+import { Template } from 'meteor/templating';
+
+import './pageNotFound.html';
+
+Template.pageNotFound.helpers({
+	errorcode() {
+		return '404';
+	},
+});
```

Comment:
I don't think you need a helper here. Since the template's name is `pageNotFound`, it will be only applied for HTTP 404 cases.